src/index.js

```diff
@@ -1,2 +1,9 @@
 import "./styles/styles.scss";
 export * from "./components/TestComp.jsx";
+export { default as DatePickerCarbon } from "./components/DatePickerCarbon/DatePickerCarbon.jsx";
```

Comment:
You know, it is much easier to just have named exports everywhere. default exports in their files are a pain 😆

Reply:
I tried to do that like how you have done for the TestComp. But, named exports were not working surprisingly

Reply:
that's weird... you sure your component was being exported as a named export too?

Reply:
Yes, It was giving this error even though the exports were added
 
 ```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Wr`.
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Wr`.
    at createFiberFromTypeAndProps (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:23965:21)
    at createFiberFromElement (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:23988:15)
    at reconcileSingleElement (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:14233:23)
    at reconcileChildFibers (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:14293:35)
    at reconcileChildren (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:16762:28)
    at updateHostComponent (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:17302:3)
    at beginWork (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:18627:14)
    at HTMLUnknownElement.callCallback (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:188:14)
    at Object.invokeGuardedCallbackDev (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:237:16)
    at invokeGuardedCallback (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:292:31)
```

Reply:
Yup, added named exports in component and export like how it is done for TestComp in index.js. Still giving this error

```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Wr`.
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Wr`.
    at createFiberFromTypeAndProps (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:23965:21)
    at createFiberFromElement (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:23988:15)
    at reconcileSingleElement (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:14233:23)
    at reconcileChildFibers (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:14293:35)
    at reconcileChildren (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:16762:28)
    at updateHostComponent (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:17302:3)
    at beginWork (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:18627:14)
    at HTMLUnknownElement.callCallback (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:188:14)
    at Object.invokeGuardedCallbackDev (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:237:16)
    at invokeGuardedCallback (webpack://openmrs-module-ipd-frontend/./node_modules/react-dom/cjs/react-dom.development.js?:292:31)

```